### DISCUSSION TITLE
3E9A73F5-701A-2875-18F5-4899CD66A1EA

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#83] [General]
# M 81 (Bode's Galaxy)

Bode's Galaxy (also known as Messier 81 or NGC 3031) is an Sb-type grand design spiral galaxy about 12 million light-years away in the constellation Ursa Major. It is 90,000 light-years across, and in 1993, the first Type-IIb supernova ever discovered was observed in this galaxy

https://celestia.mobi/addon?item=3E9A73F5-701A-2875-18F5-4899CD66A1EA&lang=pt

**Comment by Ugu999** [comment]
Precisa Melhorar a Foto

**Comment by kmjordan122313-lgtm** [comment]
it looks weird in the thumbnail, but ok.

**Comment by kmjordan122313-lgtm** [comment]
it's kinda nice.
